src/query/settings/src/settings_default.rs

```diff
@@ -590,6 +590,15 @@ impl DefaultSettings {
                     mode: SettingMode::Both,
                     range: Some(SettingRange::Numeric(0..=1)),
                 }),
+                (
+                    "enable_compact_after_multi_table_insert",
+                    DefaultSettingValue {
+                        value: UserSettingValue::UInt64(1),
```

Comment:
let's disable it by default, since some users are expecting no auto compaction for multi-table insertions